perllib/FixMyStreet/SendReport/Email/Highways.pm

```diff
@@ -30,15 +30,15 @@ sub build_recipient_list {
 sub _lookup_site_code_config { {
     buffer => 15, # metres
     url => "https://tilma.mysociety.org/mapserver/highways",
-    srsname => "urn:ogc:def:crs:EPSG::4326",
+    srsname => "urn:ogc:def:crs:EPSG::27700",
```

Comment:
The tests are okay without changing this, as far as I can see, unaffected tests should not need to be changed, a sign something else must be wrong.